thoth.go

```diff
@@ -6,17 +6,25 @@ import (
 	"time"
 )
 
+var rng *rand.Rand
+
+// CreateRand initializes the random generator.
+func CreateRand() {
```

Comment:
make the function private ```suggestion
func createRand() {
```